cmd/mkcrl/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/json"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/cloudflare/cfssl/crypto/pkcs11key"
+	"github.com/cloudflare/cfssl/log"
+)
+
+var certFile = flag.String("ca", "", "JSON file for subject and validity")
+var listFile = flag.String("revoked", "", "JSON list of revoked cert serials")
```

Comment:
The format is not totally clear here. I think it should probably be:

`JSON list of pkix.RevokedCertificate objects (https://golang.org/pkg/crypto/x509/pkix/#RevokedCertificate)`